src/query/api/v1/handler/prometheus/common.go

```diff
@@ -191,7 +196,7 @@ func ParseSeriesMatchQuery(
 		return nil, xhttp.NewParseError(err, http.StatusBadRequest)
 	}
 
-	tagMatchers := make([]models.Matchers, len(matcherValues))
+	queries := make([]*storage.FetchQuery, len(matcherValues))
```

Comment:
on line 189 - what's with the magic numbers? (unrelated to this change)

Reply:
Was defaulting to 40 days as query start for ...reasons; changed to go from 0 until Now by default